src/main/resources/avro/io/confluent/examples/streams/pageviewregion.avsc

```diff
@@ -1,6 +1,6 @@
 {"namespace": "io.confluent.examples.streams.avro",
  "type": "record",
- "name": "PageViewStats",
+ "name": "PageViewRegion",
```

Comment:
Should it be `PageViewByRegion` ?

Reply:
I'm not sure.  The example was written in avro using `GenericRecord`, I have serialized it with plain Java serialization so found the error. I'm ok with either name.

Reply:
Don't have a strong opinion either. The output topic is called `PageViewByRegion` to it might be good to align names.

Reply:
+1 to align names properly (= use `PageViewByRegion`).

Note that this should also require updating both the Java 7 and Java 8+ variants of our accompanying examples.